### PR TITLE
sdcm/cluster: fix get_nodetool_status

### PR DESCRIPTION
**Pull request by amoskong**
Related ticket: https://trello.com/c/LK4DbYrT/1192-streaming-error-handling-scenarios-nemesis

get_nodetool_status() is used when I completed above task, so send this PR to fix the function first.

This patch fixed the split pattern of datacenter, currently all nodes status
will be wrongly merged into one datacenter. Ignored the spare title line
from result, and added the load unit to result.

```
@#### Expected return of example status:
{
   'datacenter2':{
      '172.17.0.3':{
         'load':'456.53KB',  <<<<<  load unit is added
         'tokens':'256',
         'state':'UN',
         'owns':'?',
         'host_id':'70d17de8-2839-4f30-829a-504d1a06176f',
         'rack':'rack1'
      }
   },
   'datacenter1':{
      '172.17.0.2':{
         'load':'456.53KB',
         'tokens':'256',
         'state':'UN',
         'owns':'?',
         'host_id':'70d17de8-2839-4f30-829a-504d1a06176f',
         'rack':'rack1'
      }
   }
}

@#### Actually return of example status:
{
   'Datacenter: datacenter1':{    <<<<<< wrong datacenter name
      '172.17.0.2':{
         'load':'456.53',
         'tokens':'256',
         'state':'UN',
         'owns':'?',
         'host_id':'70d17de8-2839-4f30-829a-504d1a06176f',
         'rack':'rack1'
      },
      '172.17.0.3':{
         'load':'456.53',
         'tokens':'256',
         'state':'UN',
         'owns':'?',
         'host_id':'70d17de8-2839-4f30-829a-504d1a06176f',
         'rack':'rack1'
      },
      'Address':{                   <<<<< spare title line
         'load':'Load Tokens',
         'tokens':'Owns',
         'state':'--',
         'owns':'Host',
         'host_id':'ID',
         'rack':'Rack'
      }
   }
}

Example status:

"""
Datacenter: datacenter1
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens       Owns    Host ID                               Rack
UN  172.17.0.2  456.53 KB  256          ?       70d17de8-2839-4f30-829a-504d1a06176f  rack1

Datacenter: datacenter2
=======================
Status=Up/Down
|/ State=Normal/Leaving/Joining/Moving
--  Address     Load       Tokens       Owns    Host ID                               Rack
UN  172.17.0.3  456.53 KB  256          ?       70d17de8-2839-4f30-829a-504d1a06176f  rack1

Note: Non-system keyspaces don't have the same replication settings, effective ownership information is meaningless
"""
```

Signed-off-by: Amos Kong <amos@scylladb.com>

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -2803,17 +2803,19 @@ def get_nodetool_status(self, verification_node=None):  # pylint: disable=too-ma
...
                 dc_name = lines[0]
                 status[dc_name] = {}
                 for line in lines[1:]:
+                    if line.startswith('--'):  # ignore the title line in result
+                        continue
                     try:
                         state, ip, load, _, tokens, owns, host_id, rack = line.split()
                         node_info = {'state': state,
-                                     'load': load,
+                                     'load': '%s%s' % (load, _),
```
Comment (bentsi) [inline sdcm/cluster.py]: What does it mean??? confusing

File: sdcm/cluster.py [hunk 2]
```diff
@@ -2803,17 +2803,19 @@ def get_nodetool_status(self, verification_node=None):  # pylint: disable=too-ma
             verification_node = random.choice(self.nodes)
         status = {}
         res = verification_node.run_nodetool('status')
-        data_centers = res.stdout.strip().split("Data center: ")
+        data_centers = res.stdout.strip().split("Datacenter: ")
```
Comment (bentsi) [inline sdcm/cluster.py]: how did it work before? Was the `nodetool` string changed recently?

File: sdcm/cluster.py (hunk 1 above)
Comment (amoskong) [reply sdcm/cluster.py]: unit string is assigned to `_` in the past, here I just joined them together.
Original result only contains the number without the unit string.

Comment (amoskong) [reply sdcm/cluster.py]: I will rename the `_` to `load_unit`

File: sdcm/cluster.py (hunk 2 above)
Comment (amoskong) [reply sdcm/cluster.py]: > how did it work before?

Work for lucky.
- Invalid result is ignored in counting the living node.
- we only check if some nodes are living, we don't verify which dc the node is in.

> Was the `nodetool` string changed recently?

No, the datacenter string didn't change recently. 
Question of nodetool 5 years ago: https://stackoverflow.com/questions/20678366/how-to-interpret-the-output-of-nodetool-status

**Pull request merged**